### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by Adriatic1**
I've replaced old project name with new one in README.md file. 
Additionally I've added commands to clarify how to do the clone operation.

### CODE REVIEW

**Comment by nacho** [comment]
I don't think this is correct, you still need the github subdirectory no?

**Comment by Adriatic1** [comment]
It works OK for me without github subdirectory.

**Comment by guruDanny67** [comment]
Actually you can clone the project in any directory you like and, out of the box, it works: the working dirs (build, tools) are created from the main build dir (c:\gtk-build) and the patches source dir is now calculated from the script.

File: README.md [hunk 1]
```diff
@@ -29,19 +29,25 @@ HexChat developers decided that their script should focus on their specific need
     pacman -S nasm patch gettext make coreutils diffutils wget yasm pkg-config
     ```
 
-1. Clone [this repository](https://github.com/wingtk/gtk-win32) to _C:\gtk-build\github\gtk-win32_ It contains the build script, project files and patches.
+1. Clone [this repository](https://github.com/wingtk/gvsbuild) to _C:\gtk-build\gvsbuild_ It contains the build script, project files and patches.
```
Comment (nacho) [inline README.md]: Dunno I guess we can change the instructions to just: "Clone [this repository](https://github.com/wingtk/gvsbuild). It contains the build script, project files and patches.

Comment (Adriatic1) [inline README.md]: IMO, it is important to have precise instructions, especially for people who not experienced with git. Old instructions left unclear if the git clone will create github folder as well. That's why I would prefer the steps to be more precise this time.

Comment (nacho) [inline README.md]: I agree they should be precise but currently there is no need to create the github directory. So the instructions are:
git clone repo
cd repo
python build.py build gtk3

**Comment by Adriatic1** [comment]
OK, agreed. I just hope it won't be done within "C:\" root :-) (although it is possible).

**Comment by Adriatic1** [comment]
You can close this one.
As I see, since this pull request, you already replaced old project name with new one.

**Pull request closed without merging**